WeatherToday/Sources/Domain/UseCases/WeatherUseCase.swift

```diff
@@ -0,0 +1,24 @@
+//
+//  WeatherUseCase.swift
+//  WeatherToday
+//
+//  Created by Taher on 22/10/22.
+//  Copyright © 2022 Nimble. All rights reserved.
+//
+
+import Alamofire
+import RxCocoa
+import RxSwift
+
+final class WeatherUseCase: WeatherUseCaseProtocol {
```

Comment:
FYI: A use case should begin with a verb, so we should use the name `GetWeatherUseCase`, and its function for getting weather should be `execute`

```suggestion
final class GetWeatherUseCase: GetWeatherUseCaseProtocol {

  func execute(forCity name: String) -> Single<WeatherApi> {
      weatherRepository.getWeatherData(cityName: name)
  }
}
```

Reply:
updated in this [commit](https://github.com/nimblehq/ci-cd-sample-ios/pull/23/commits/c01b2420a65d92ab54c11dac43f4a1a23d2adece)